modules/images-imagestream-import-images-private-registry.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// * assembly/openshift_images/managing-image-streams.adoc
+
+[id="images-imagestream-import-images-private-registry_{context}"]
+= Importing images from private registries
+
+An imagestream can be configured to import tag and image metadata from private image registries, requiring authentication.
+
+[NOTE]
+====
+When importing from an insecure registry, the registry URL defined in the Secret must include the :80 port suffix or the Secret is not used when attempting to import from the registry.
+====
+
+.Procedure
+
+. You must create a Secret object that is used to store your credentials by entering the following command:
++
+----
+$ oc create secret generic <secret_name> \
+    --from-file=<file>.txt
+
+secret/<secret_name>
+----
++
+. After the Secret is configured, create the new imagestream or enter the `oc import-image` command:
++
+----
+$ oc import-image <image-name>
```

Comment:
When create a new imagestream, we could use cmds:
`oc tag <image> <imagestreamtag>`
or 
`oc import-image <imagestreamtag> --from=<image> --confirm`